### ISSUE TITLE
importing lightcurve and net under Python 2 ends the world

### ISSUE THREAD

**Issue by Cadair** [sunpy/sunpy#2122]
```python
 ❯ ipython

In [1]: from sunpy.lightcurve import GOESLightCurve
   ...: from sunpy.net import hek
   ...: 
WARNING: AstropyDeprecationWarning: As of v0.8.0, the `sunpy.wcs` module is deprecated and will be removed in a future version. Use `sunpy.coordinates` or `sunpy.map` for coordinate transformations. [sunpy.wcs]
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-913dc430db3e> in <module>()
      1 from sunpy.lightcurve import GOESLightCurve
----> 2 from sunpy.net import hek

/home/stuart/Git/sunpy/sunpy/net/__init__.py in <module>()
      3 __all__ = ["Fido"]
      4 
----> 5 from sunpy.net.fido_factory import Fido

/home/stuart/Git/sunpy/sunpy/net/fido_factory.py in <module>()
     21 from .dataretriever.client import QueryResponse
     22 from .vso import VSOClient
---> 23 from . import attr
     24 from . import attrs as a
     25 

ImportError: cannot import name attr
```
However the following works:
```python
 ❯ ipython

In [1]: from sunpy.net import hek
WARNING: AstropyDeprecationWarning: As of v0.8.0, the `sunpy.wcs` module is deprecated and will be removed in a future version. Use `sunpy.coordinates` or `sunpy.map` for coordinate transformations. [sunpy.wcs]

In [2]: from sunpy.lightcurve import GOESLightCurve
```

I don't even know who I am any more.

**Comment by aringlis** [comment]
Weird - I get the same error, but if you swap the order of the commands, it works fine:

    ln [1]: from sunpy.net import hek
    ln [2]: from sunpy.lightcurve import GOESLightCurve
    In [3]:

**Comment by Cadair** [comment]
@aringlis yeah...

It also seems to happen with `sunpy.timeseries` as well.

**Comment by Cadair** [comment]
```python
>>> import sunpy.timeseries
WARNING: AstropyDeprecationWarning: As of v0.8.0, the `sunpy.wcs` module is deprecated and will be removed in a future version. Use `sunpy.coordinates` or `sunpy.map` for coordinate transformations. [sunpy.wcs]
>>> from sunpy.net import hek
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/stuart/Git/sunpy/sunpy/net/__init__.py", line 5, in <module>
    from sunpy.net.fido_factory import Fido
  File "/home/stuart/Git/sunpy/sunpy/net/fido_factory.py", line 23, in <module>
    from . import attr
ImportError: cannot import name attr
```

**Comment by Cadair** [comment]
I am therefore assuming that it's something in one of the old lightcurve source files.

**Comment by nabobalis** [comment]
Please hold.

**Comment by nabobalis** [comment]
While this is a bug, I don't think we will fix it and with master now 3.6+, I will close this.
